src/hooks/__tests__/useMessageQueue.test.ts

```
@@ -0,0 +1,474 @@
+  }
+
+  const runEffects = () => {
+    refreshAtomMock()
+    // Re-evaluate the hook to capture fresh effects
+    ;(useEffect as jest.Mock).mockImplementation((fn: () => void) => {
+      effects.length = 0
+      effects.push(fn)
+    })
+    // eslint-disable-next-line react-hooks/rules-of-hooks -- test helper
+    const freshResult = useMessageQueue(mocks)
+    effects.forEach((fn) => fn())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)